python/example_code/lookoutvision/export_datasets.py

```diff
@@ -0,0 +1,271 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Purpose
+
+Shows how to export the datasets (manifest files and images)
+from an Amazon Lookout for Vision project to a new S3 location.
+"""
+
+import argparse
+import json
+import logging
+
+import boto3
+from botocore.exceptions import ClientError
+
+logger = logging.getLogger(__name__)
+
+
+def copy_file(s3_resource, source_file,  destination_file):
+    """
+    Copies a file from a source S3 folder to a destination S3 folder.
+    The destination can be in different bucket.
+    :param s3: An S3 boto resource.
+    :param source_file: The S3 path to the source file.
+    :param destination_file: The destination S3 path for the copy operation.
```

Comment:
All "S3" uses have to be "Amazon S3" except for specific cases.

Reply:
All "S3" uses should be "Amazon S3", unless when referencing "S3 bucket".

Reply:
Fixed.